packages/php-db-import-export/src/Backend/SourceDestinationColumnMap.php

```diff
@@ -18,39 +18,64 @@
  */
 final class SourceDestinationColumnMap
 {
+    /**
+     * This mode will keep order of columns in both tables
+     * Map will fail if columns count is different
+     */
+    public const MODE_MAP_BY_ORDER = 'order';
+    /**
+     * This mode will take source order and map destination columns by name
+     * Map will fail if column is missing on one of the tables
+     */
+    public const MODE_MAP_BY_NAME = 'name';
+
```

Comment:
Nechal sem původní chování, rád bych to rozšířil na Synapse a pak na Assert který porovnává dvě tabulky, je tam cca stejná logika, ale tady to je univerzálnější.